src/mach/load_command.rs

```diff
@@ -1014,6 +1015,7 @@ pub struct VersionMinCommand {
 }
 
 impl VersionMinCommand {
+    // TODO: Needs a discriminant for TvOS, WatchOS.
```

Comment:
Yeah, we ought to break this interface. Maybe add a new `#[non_exhaustive] enum Platform` and add a `fn platform(&self) -> Platform` while we're at it.

Reply:
SGTM! I can make that part of this PR.